packages/extension-ui/src/components/themes.ts

```diff
@@ -93,6 +93,7 @@ const darkTheme = {
   readonlyInputBackground: '#1A1B20',
   secondaryFontFamily: 'Gilroy, sans-serif',
   signingBorderColor: '#7A8B99',
+  neuterColor: '#7A8B99',
```

Comment:
I didn't know the word `netuer`, but looks like it translates to _bezpłciowy_ 😅 How about just `neutral` instead? 😄

Reply:
hahaha, I seems it wasn't a good idea to translate `nijaki` to english... my bad :laughing: